events/pull-nodeschool.js

```diff
@@ -28,11 +31,11 @@ request('http://nodeschool.io/chapters/list.json', {json: true}, function (e, re
   })
 
   function _geo () {
-    if (chapters.length === 0) return yml.save()
-    var chapter = chapters.shift()
-    geocoder.geocode(chapter.location, function (err, res) {
+    if (chapters.length === 0) { return yml.save() }
```

Comment:
You are right, there isn't a consistent style for this, not even in this diff.
Forget the above comment.